imagery/i.group/i.group.md

```diff
@@ -29,6 +29,58 @@ only the visible light bands of Landsat-7:
 i.group group=vis_bands subgroup=vis_bands input=lsat7_2000_10,lsat7_2000_20,lsat7_2000_30
 ```
 
+To list files in the `vis_bands` subgroup under the `vis_bands` group:
+
+```sh
+# In shell format
+i.group group=vis_bands subgroup=vis_bands -l format=shell
+lsat7_2000_10@PERMANENT
+lsat7_2000_20@PERMANENT
+lsat7_2000_30@PERMANENT
+
+# In JSON format
+i.group group=vis_bands subgroup=vis_bands -l format=json
+[
+    "lsat7_2000_10@PERMANENT",
+    "lsat7_2000_20@PERMANENT",
+    "lsat7_2000_30@PERMANENT"
+]
+```
+
+To list maps in the `vis_bands` group using Python:
+
+```python
+import grass.script as gs
+
+# Run i.group with JSON output and the -l flag
+data = gs.parse_command(
+    "i.group",
+    group="vis_bands",
+    flags="l",
+    format="json",
+)
+
+print(data)
+```
+
+```text
+['lsat7_2000_10@PERMANENT', 'lsat7_2000_20@PERMANENT', 'lsat7_2000_30@PERMANENT']
+```
+
+To list subgroups in the `vis_bands` group:
+
+```sh
+# In shell format
+i.group group=vis_bands -s format=shell
+vis_bands
+
+# In JSON format
+i.group group=vis_bands -s format=json
+[
+    "vis_bands"
+]
+```
```

Comment:
Let's have the command and output as separate code blocks, and make the json example with Python.